packages/cli/src/ui/hooks/useShellHistory.test.ts

```diff
@@ -112,17 +117,25 @@ describe('useShellHistory', () => {
 
     // History is loaded newest-first: ['cmd2', 'cmd1']
     expect(command).toBe('cmd2');
+
+    await act(async () => {
+      unmount();
```

Comment:
Should we just wrap unmount from renderHook in act?

Reply:
reverted all these. this change is from before I did that. tried to cleanup all the cases but missed a few.